odc/stats/_text.py

```diff
@@ -81,6 +81,22 @@ def parse_yaml_file_or_inline(s: str) -> Dict[str, Any]:
     return result
 
 
+def load_yaml_remote(yaml_url: str) -> Dict[str, Any]:
+    """
+       Open a yaml file remotely and return the parsed yaml document 
+    """
+    import urllib
+    import fsspec
```

Comment:
`fsspec` should be added to `install_requires` in `setup.cfg`. Hopefully adding this exception doesn't cause any troubles, but there's a chance it will due to `aiobotocore`.